### PR TITLE
Fix jballerina unit tests

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
Fixes `TypeCastExprTest` and `PackageImportTest`.

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/typecast/TypeCastExprTest.java [hunk 1]
```diff
@@ -529,7 +529,7 @@ public void testMapToAny() {
     @Test(description = "Test casting a struct to another struct in a different package")
     public void testCastToStructInDifferentPkg() {
 //        CompileResult res = BCompileUtil.compile("test-src/expressions.typecast.foo", "expressions.typecast.foo");
```
Comment (hemikak) [inline tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/typecast/TypeCastExprTest.java:531]: Shall we remove this line ?

File: tests/jballerina-unit-test/src/test/resources/test-src/expressions.typecast.foo/Ballerina.toml [hunk 2]
```diff
@@ -1,3 +1,4 @@
-[project]
-org-name = "ballerina-test"
+[package]
+org = "test_org"
+name = "typecast"
 version = "0.0.0"
```
Comment (hemikak) [inline tests/jballerina-unit-test/src/test/resources/test-src/expressions.typecast.foo/Ballerina.toml:4]: Shall we add a new line here ?

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/typecast/TypeCastExprTest.java (hunk 1 above)
Comment (NipunaRanasinghe) [reply tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/typecast/TypeCastExprTest.java]: +1. Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/26802/commits/f6aa23950cf37c29926c3e48d1cc2492591fb017

File: tests/jballerina-unit-test/src/test/resources/test-src/expressions.typecast.foo/Ballerina.toml (hunk 2 above)
Comment (NipunaRanasinghe) [reply tests/jballerina-unit-test/src/test/resources/test-src/expressions.typecast.foo/Ballerina.toml]: +1. Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/26802/commits/f6aa23950cf37c29926c3e48d1cc2492591fb017

**Review by hemikak — APPROVED** [review]

**Pull request merged**